test/backend/datastores.mock.spec.js

```diff
@@ -0,0 +1,183 @@
+import {assert} from 'chai';
+
+import {saveConnection} from '../../backend/persistent/Connections.js';
+import {saveSetting} from '../../backend/settings.js';
+
+import {
+    assertResponseStatus,
+    clearSettings,
+    closeTestServers,
+    createTestServers,
+    getResponseJson,
+    POST,
+    sqlConnections
+} from './utils.js';
+
+
+let servers;
+let connectionId;
+
+describe('Datastore Mock:', function () {
+    beforeEach(() => {
+        servers = createTestServers();
+        saveSetting('AUTH_ENABLED', false);
+        sqlConnections.mock = true;
+        connectionId = saveConnection(sqlConnections);
+    });
+
+    afterEach(() => {
+        clearSettings('SETTINGS_PATH');
+        return closeTestServers(servers);
+
```

Comment:
:goat:

Reply:
what does 🐐 mean here?

Reply:
I'm not sure either.

For reference, here's the list: https://github.com/plotly/dev-docs/blob/master/basics/code-review-checklist.md#code-review-animals

Reply:
:space_invader:

Reply:
This is not _too_ much space IMHO. I am not sure if any Javascript guidelines mandates to not leave any blank lines in code. Can you please point to the relevant document..

Reply:
:disappointed: Let's not waste our time discussing over white space. There are more interesting things to fix in this PR.